packages/@aws-cdk/core/lib/private/construct-tree-metadata.ts

```diff
@@ -0,0 +1,55 @@
+import fs = require('fs');
+import path = require('path');
+
+import { ArtifactType } from '@aws-cdk/cx-api';
+import { Construct, IConstruct, ISynthesisSession } from '../construct';
+
+const FILE_PATH = 'construct-tree-metadata.json';
```

Comment:
I preferred “tree.json”. Generally trying to keep naming concise.

Reply:
tree.json is a bit too concise IMO. How about `construct-tree.json` and `ConstructTree` instead?

Reply:
Not sure what value the word “construct” adds in this context. The cloud assembly is an artifact of a cdk app which is composed of constructs by definition. “tree” conveys this quite well.

Reply:
Couldn't we possibly publish other things in the form of trees? eg. a tree of CF stacks when one stack depends on the other, a tree form of assets. Essentially anything modeled as a hierarchy is a tree. Adding the word 'construct' clarifies this.

Further, wrt the CloudAssembly manifest, I find the former more self-descriptive than the latter, while staying terse.

```json
"ConstructTree": {
    "type": "cdk:metadata"
```

```json
"Tree": {
    "type": "cdk:metadata"
```

Reply:
Continued this conversation with Elad over chat. Switching to use Tree instead of ConstructTree.